playwright/manual-single.test.ts

```diff
@@ -0,0 +1,55 @@
+import { expect, test } from '@playwright/test';
+
+const viewport = { width: 1600, height: 1000 };
+
+const widths = ['360', '740', '980', '1300', '100%'];
```

Comment:
Minor - so feel free to ignore. We could get breakpoints from [source-foundations](https://github.com/guardian/source/blob/d84c25d67ed51545994f53986eb4487d35c1aab4/packages/%40guardian/source-foundations/src/breakpoints/breakpoints.ts#L15)

Reply:
I think because of the 100% value in there not being in source I might leave this for now. But I have extracted the widths into a shared const instead so that they're only defined in one place. I've also updated the variable naming to make it a bit clearer what these sizes actually represent